src/js/node/http.ts

```diff
@@ -1249,18 +1249,18 @@ export class ClientRequest extends OutgoingMessage {
       callback();
       return;
     }
-    this.#body = body + chunk;
+    this.#body = Buffer.concat([body, chunk]);
```

Comment:
I see that the `chunk` has already been converted to the `Buffer` type here.

https://github.com/oven-sh/bun/blob/7a8f57c4e527f7fc84b85a4757723bb97540ebc3/src/js/node/stream.js#L3573-L3589

Reply:
This will make node:http a lot slower.

Can we avoid doing this when the input is a string or properly-encoded string? Maybe we can use BufferList for this?

Reply:
Fixed.